test/e2e/plain_provisioner_test.go

```diff
@@ -1186,6 +1187,66 @@ var _ = Describe("plain provisioner bundleinstance", func() {
 			))
 		})
 	})
+
+	When("a BundleInstance targets a registry+v1 Bundle", func() {
+		var (
+			bi  *rukpakv1alpha1.BundleInstance
+			ctx context.Context
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			bi = &rukpakv1alpha1.BundleInstance{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "prometheus",
+				},
+				Spec: rukpakv1alpha1.BundleInstanceSpec{
+					ProvisionerClassName: plain.ProvisionerID,
+					Template: &rukpakv1alpha1.BundleTemplate{
+						ObjectMeta: metav1.ObjectMeta{
+							Labels: map[string]string{
+								"app.kubernetes.io/name": "prometheus",
+							},
+						},
+						Spec: rukpakv1alpha1.BundleSpec{
+							ProvisionerClassName: registry.ProvisionerID,
+							Source: rukpakv1alpha1.BundleSource{
+								Type: rukpakv1alpha1.SourceTypeImage,
+								Image: &rukpakv1alpha1.ImageSource{
+									Ref: "quay.io/operatorhubio/prometheus:v0.47.0--20220325T220130",
+								},
+							},
+						},
+					},
+				},
+			}
+			err := c.Create(ctx, bi)
+			Expect(err).To(BeNil())
+		})
+		AfterEach(func() {
+			By("deleting the testing BI resource")
+			Expect(c.Delete(ctx, bi)).To(BeNil())
+		})
+
+		It("should rollout the bundle contents successfully", func() {
```

Comment:
I thought this might be enough for #427.  The granular tests may be coved in #430.  WDYT?

Reply:
I'm not sure this closes #427 but I think it's fine for #430 (for now). For #427 I think we want to do some more indepth checks around auth (where if you don't have the right permissions, writing to that endpoint would fail?)